bin/test-workflow/dev-start.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# This script is intended for use by developers and not to be called by automation.
+
+set -euo pipefail
+IFS=$'\n\t'
+export GCLOUD_PROJECT_NAME="${GCLOUD_PROJECT_NAME:-gke}"
+export GCLOUD_ZONE="${GCLOUD_ZONE:-gke}"
+export GCLOUD_CLUSTER_NAME="${GCLOUD_CLUSTER_NAME:-gke}"
+export GCLOUD_SERVICE_KEY="${GCLOUD_SERVICE_KEY:-gke}"
+
+print_usage() {
+  echo "Usage:"
+  echo "    This script will run start for the various platforms"
+  echo ""
+  echo "Syntax:"
+  echo "    $0 [Options]"
+  echo "    Options:"
+  echo "    -c                         Run in a container"
```

Comment:
Maybe clearer to say something like "Run clients from a container"? My initial read of this was thinking this could be a KinD or Docker-in-Docker setup.

Reply:
good point, updated.